synth/validator/miner_data_handler.py

```diff
@@ -164,6 +166,13 @@ def get_latest_prediction_request(
                             * validator_requests.c.time_length
                         )
                         < scored_time,
+                        scored_time
+                        - (
+                            validator_requests.c.start_time
+                            + text("INTERVAL '1 second'")
+                            * validator_requests.c.time_length
+                        )
+                        < timedelta(seconds=DELAY_BETWEEN_ITERATIONS),
```

Comment:
one hour gap, i.e. it looks for `start_time >= scored_time - 24h - 1h` should be fine, but better to run this code for a couple of days on the testnet to doublecheck that no records are missing

Reply:
yes and also it doesn't fix entirely the issue: https://linear.app/mode/issue/MOD-1357/fix-duplication-issue-for-miner-predictions#comment-2aadd9c6